test/parallel_testsuite.py

```diff
@@ -62,11 +67,18 @@ def run(self, result):
     # issues.
     # multiprocessing.set_start_method('spawn')
     tests = list(self.reversed_tests())
+    contains_browser_test = any(test.is_browser_test() for test in tests)
     use_cores = cap_max_workers_in_pool(min(self.max_cores, len(tests), num_cores()))
+    if contains_browser_test:
+      # TODO experiment with this number. In browser tests we'll be creating
+      # a chrome instance per worker which is expensive.
+      use_cores = int(use_cores / 2)
```

Comment:
It seems like this limit should not be applied the result of `num_cores` since it would be nice if `EMCC_CORES` or `EMTEST_CORES` or `-j` was honored precisely.  

So maybe this logic should go inside the `cap_max_workers_in_pool` function it should take a "is_browser" 4th argument?